tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -79,7 +79,9 @@ def _run(self):
                         lock.acquire(lock_ttl=60)
                 except etcd.EtcdLockExpired:
                     continue
-                gevent.sleep(10)
+                gevent.sleep(
+                    int(NS.config.data.get("sync_interval", 10))
```

Comment:
Should I grep for ttl and whenever there is ttl being set for the entities, change it to sync_interval + 15 ?